tarodb/sqlc/migrations/000002_assets.up.sql

```diff
@@ -31,6 +31,21 @@ CREATE TABLE IF NOT EXISTS genesis_points (
     anchor_tx_id INTEGER REFERENCES chain_txns(txn_id)
 );
 
+-- assets_meta is a table that holds all the metadata information for genesis
+-- assets that we either created, or bootstrapped from the relevant Base
+-- Universe.
+CREATE TABLE IF NOT EXISTS assets_meta (
+    meta_id INTEGER PRIMARY KEY,
+
+    meta_data_hash BLOB CHECK(length(meta_data_hash) = 32) UNIQUE,
```

Comment:
I don't think this matches the behavior on the Golang side.

IIUC, we allowed empty metadata before - `[]byte{}` gets stored as `NULL`, which was allowed before.

Now I think an unset metadata hash would be either `[]byte{}` or all 0's. But I don't think we can use the zero hash across multiple assets because of the `UNIQUE` constraint?

I think maybe we'd want to map both []byte{} and the zero hash to NULL.

Reply:
It would be a `[32]byte`, I pushed a modified version of one of the commits to test this logic explicitly.